diff -Nuard control-center-2.4.0.orig/configure.in control-center-2.4.0/configure.in
--- control-center-2.4.0.orig/configure.in	2003-09-08 16:55:21.000000000 +0200
+++ control-center-2.4.0/configure.in	2003-11-08 23:39:16.000000000 +0100
@@ -330,7 +330,7 @@
 AC_DEFINE_UNQUOTED(GNOMECC_DATA_DIR, "${real_prefix}/share/${GETTEXT_PACKAGE}",[Path to gnomecc data files])
 GNOMECC_ICONS_DIR='$(datadir)/'${GETTEXT_PACKAGE}'/icons'
 AC_SUBST(GNOMECC_ICONS_DIR)
-GNOMECC_DESKTOP_DIR='$(datadir)/'${GETTEXT_PACKAGE}'/capplets'
+GNOMECC_DESKTOP_DIR='$(datadir)/gnome/capplets'
 AC_SUBST(GNOMECC_DESKTOP_DIR)
 GNOMECC_GLADE_DIR='$(datadir)/'${GETTEXT_PACKAGE}'/interfaces'
 AC_SUBST(GNOMECC_GLADE_DIR)
