--- capplets/default-applications/gnome-default-applications-properties-structs.c.orig	2004-04-07 06:31:42.000000000 +0200
+++ capplets/default-applications/gnome-default-applications-properties-structs.c	2004-08-13 09:11:34.868186984 +0200
@@ -9,8 +9,7 @@
         { N_("Epiphany"), 		"epiphany",	"epiphany %s",		FALSE, FALSE },
         { N_("Galeon"), 		"galeon",	"galeon %s",		FALSE, FALSE },
         { N_("Encompass"), 		"encompass",	"encompass %s",		FALSE, FALSE },
-        { N_("Firebird/FireFox"),  "mozilla-firebird",	"mozilla-firebird %s",	FALSE, FALSE },
-        { N_("Mozilla/Netscape 6"), 	"mozilla-1.6",	"mozilla-1.6 %s",	FALSE, FALSE },
+        { N_("Firebird/FireFox"),	"mozilla-firefox","mozilla-firefox %s",	FALSE, FALSE },
         { N_("Mozilla/Netscape 6"), 	"mozilla",	"mozilla %s",		FALSE, FALSE },
         { N_("Netscape Communicator"), 	"netscape",	"netscape %s",		FALSE, FALSE },
         { N_("Konqueror"), 		"konqueror",	"konqueror %s",		FALSE, FALSE },
