--- control-center-2.5.0/capplets/font/main.c.wiget	2003-10-28 22:56:03.000000000 +0100
+++ control-center-2.5.0/capplets/font/main.c	2003-11-14 12:15:59.479052850 +0100
@@ -10,6 +10,7 @@
 #include <glade/glade.h>
 #include <stdarg.h>
 #include <math.h>
+#include <fontconfig/fontconfig.h>
 
 #ifdef HAVE_XFT2
 #include <gdk/gdkx.h>
--- control-center-2.5.0/control-center/main.c.wiget	2002-05-16 18:38:03.000000000 +0200
+++ control-center-2.5.0/control-center/main.c	2003-11-14 12:14:25.849339822 +0100
@@ -32,6 +32,7 @@
 #include <X11/Xlib.h>
 #include <X11/Xatom.h>
 #include <gdk/gdkx.h>
+#include <fontconfig/fontconfig.h>
 
 #include "capplet-dir.h"
 #include "capplet-dir-view.h"
