diff -Nru control-center-1.0.53/capplets/background-properties/Makefile.am control-center-1.0.53.new/capplets/background-properties/Makefile.am
--- control-center-1.0.53/capplets/background-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/background-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -23,4 +23,4 @@
 sys_DATA = background-properties.desktop 
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/background-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/Desktop/background-properties.desktop
+	$(INSTALL_DATA) $(srcdir)/background-properties.desktop $(DESTDIR)$(datadir)/applnk/Settings/Desktop/background-properties.desktop
diff -Nru control-center-1.0.53/capplets/gnome-edit-properties/Makefile.am control-center-1.0.53.new/capplets/gnome-edit-properties/Makefile.am
--- control-center-1.0.53/capplets/gnome-edit-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/gnome-edit-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -21,4 +21,4 @@
 sys_DATA = gnome-edit-properties.desktop
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/gnome-edit-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/gnome-edit-properties.desktop
+	$(INSTALL_DATA) $(srcdir)/gnome-edit-properties.desktop $(DESTDIR)$(datadir)/applnk/Settings/gnome-edit-properties.desktop
diff -Nru control-center-1.0.53/capplets/keyboard-properties/Makefile.am control-center-1.0.53.new/capplets/keyboard-properties/Makefile.am
--- control-center-1.0.53/capplets/keyboard-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/keyboard-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -19,4 +19,4 @@
 sys_DATA = keyboard-properties.desktop
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/keyboard-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/Peripherals/keyboard-properties.desktop
+	$(INSTALL_DATA) $(srcdir)/keyboard-properties.desktop $(DESTDIR)$(datadir)/applnk/Settings/Peripherals/keyboard-properties.desktop
diff -Nru control-center-1.0.53/capplets/mime-type/Makefile.am control-center-1.0.53.new/capplets/mime-type/Makefile.am
--- control-center-1.0.53/capplets/mime-type/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/mime-type/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -23,4 +23,4 @@
 sys_DATA = mime-type.desktop 
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/mime-type.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/mime-type.desktop
+	$(INSTALL_DATA) $(srcdir)/mime-type.desktop $(DESTDIR)$(datadir)/applnk/Settings/mime-type.desktop
diff -Nru control-center-1.0.53/capplets/mouse-properties/Makefile.am control-center-1.0.53.new/capplets/mouse-properties/Makefile.am
--- control-center-1.0.53/capplets/mouse-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/mouse-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -20,4 +20,4 @@
 sys_DATA = mouse-properties.desktop 
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/mouse-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/Peripherals/mouse-properties.desktop
+	$(INSTALL_DATA) $(srcdir)/mouse-properties.desktop $(DESTDIR)$(datadir)/applnk/Settings/Peripherals/mouse-properties.desktop
diff -Nru control-center-1.0.53/capplets/screensaver-properties/Makefile.am control-center-1.0.53.new/capplets/screensaver-properties/Makefile.am
--- control-center-1.0.53/capplets/screensaver-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/screensaver-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -29,4 +29,4 @@
 sys_DATA = screensaver-properties.desktop
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/screensaver-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/Desktop/screensaver-properties.desktop
+	$(INSTALL_DATA) $(srcdir)/screensaver-properties.desktop $(DESTDIR)$(datadir)/applnk/Settings/Desktop/screensaver-properties.desktop
diff -Nru control-center-1.0.53/capplets/session-properties/Makefile.am control-center-1.0.53.new/capplets/session-properties/Makefile.am
--- control-center-1.0.53/capplets/session-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/session-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -38,7 +38,7 @@
 sys_DATA = session-properties.desktop
 endif
 
-settingsdir = $(datadir)/gnome/apps/Settings/Session
+settingsdir = $(datadir)/applnk/Settings/Session
 
 if SESSION
 settings_DATA = session-properties.desktop
diff -Nru control-center-1.0.53/capplets/sound-properties/Makefile.am control-center-1.0.53.new/capplets/sound-properties/Makefile.am
--- control-center-1.0.53/capplets/sound-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/sound-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -21,4 +21,4 @@
 sys_DATA = sound-properties.desktop
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/sound-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/Multimedia/sound-properties.desktop
+	$(INSTALL_DATA) $(srcdir)/sound-properties.desktop $(DESTDIR)$(datadir)/applnk/Settings/Multimedia/sound-properties.desktop
diff -Nru control-center-1.0.53/capplets/theme-switcher/Makefile.am control-center-1.0.53.new/capplets/theme-switcher/Makefile.am
--- control-center-1.0.53/capplets/theme-switcher/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/theme-switcher/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -31,4 +31,4 @@
 sys_DATA = theme-selector.desktop
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/theme-selector.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/Desktop/theme-selector.desktop
+	$(INSTALL_DATA) $(srcdir)/theme-selector.desktop $(DESTDIR)$(datadir)/applnk/Settings/Desktop/theme-selector.desktop
diff -Nru control-center-1.0.53/capplets/ui-properties/Makefile.am control-center-1.0.53.new/capplets/ui-properties/Makefile.am
--- control-center-1.0.53/capplets/ui-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/ui-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -27,6 +27,6 @@
 	mdi.desktop 
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/application.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/UIOptions/application.desktop
-	$(INSTALL_DATA) $(srcdir)/dialogs.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/UIOptions/dialogs.desktop
-	$(INSTALL_DATA) $(srcdir)/mdi.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/UIOptions/mdi.desktop
+	$(INSTALL_DATA) $(srcdir)/application.desktop $(DESTDIR)$(datadir)/applnk/Settings/UIOptions/application.desktop
+	$(INSTALL_DATA) $(srcdir)/dialogs.desktop $(DESTDIR)$(datadir)/applnk/Settings/UIOptions/dialogs.desktop
+	$(INSTALL_DATA) $(srcdir)/mdi.desktop $(DESTDIR)$(datadir)/applnk/Settings/UIOptions/mdi.desktop
diff -Nru control-center-1.0.53/capplets/url-properties/Makefile.am control-center-1.0.53.new/capplets/url-properties/Makefile.am
--- control-center-1.0.53/capplets/url-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/url-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -21,4 +21,4 @@
 sys_DATA = url-properties.desktop
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/url-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/url-properties.desktop
+	$(INSTALL_DATA) $(srcdir)/url-properties.desktop $(DESTDIR)$(datadir)/applnk/Settings/url-properties.desktop
diff -Nru control-center-1.0.53/capplets/wm-properties/Makefile.am control-center-1.0.53.new/capplets/wm-properties/Makefile.am
--- control-center-1.0.53/capplets/wm-properties/Makefile.am	Fri Mar 31 02:53:59 2000
+++ control-center-1.0.53.new/capplets/wm-properties/Makefile.am	Fri Mar 31 02:53:36 2000
@@ -25,5 +25,5 @@
 sysdir = $(datadir)/control-center/Desktop
 sys_DATA = wm-properties.desktop
 
-Setdir = $(datadir)/gnome/apps/Settings/Desktop
+Setdir = $(datadir)/applnk/Settings/Desktop
 Set_DATA = wm-properties.desktop
diff -Nru control-center-1.0.53/control-center/Makefile.am control-center-1.0.53.new/control-center/Makefile.am
--- control-center-1.0.53/control-center/Makefile.am	Wed Mar 29 06:35:39 2000
+++ control-center-1.0.53.new/control-center/Makefile.am	Fri Mar 31 02:54:42 2000
@@ -42,7 +42,7 @@
  
 EXTRA_DIST = gnomecc.desktop cappletConf.sh.in $(appicon_DATA)
 
-sysdir = $(datadir)/gnome/apps/Settings
+sysdir = $(datadir)/applnk/Settings
 sys_DATA = gnomecc.desktop
 
 ## Put `exec' in the name because this should be installed by
diff -Nru control-center-1.0.53/capplets/desktop-links/Makefile.am control-center-1.0.53.new/capplets/desktop-links/Makefile.am
--- control-center-1.0.53/capplets/desktop-links/Makefile.am	Wed Mar 29 06:35:39 2000
+++ control-center-1.0.53.new/capplets/desktop-links/Makefile.am	Fri Mar 31 03:08:29 2000
@@ -5,7 +5,7 @@
 Peripheralsdir  = $(ccenterdir)/Peripherals
 Sessiondir	= $(ccenterdir)/Session
 
-paneldir        = $(datadir)/gnome/apps/Settings
+paneldir        = $(datadir)/applnk/Settings
 Multimediadirp	= $(paneldir)/Multimedia
 Desktopdirp	= $(paneldir)/Desktop
 UIOptionsdirp   = $(paneldir)/UIOptions
