diff -urN aaa/control-center-2.1.0/capplets/theme-switcher/Makefile.am control-center-2.1.0/capplets/theme-switcher/Makefile.am
--- aaa/control-center-2.1.0/capplets/theme-switcher/Makefile.am	Mon Aug 19 21:04:48 2002
+++ control-center-2.1.0/capplets/theme-switcher/Makefile.am	Tue Oct  1 21:09:46 2002
@@ -2,7 +2,7 @@
 
 gnome_theme_properties_LDADD = 						\
 	$(GNOMECC_CAPPLETS_LIBS) 					\
-	-L$(top_builddir)/libwindow-settings/ -lgnome-window-settings
+	$(top_builddir)/libwindow-settings/.libs/libgnome-window-settings.so
 
 gnome_theme_properties_SOURCES = \
 	theme-switcher.c

