diff -Nuard control-center-2.4.0.orig/capplets/default-applications/gnome-default-applications-properties.c control-center-2.4.0/capplets/default-applications/gnome-default-applications-properties.c
--- control-center-2.4.0.orig/capplets/default-applications/gnome-default-applications-properties.c	2003-08-01 21:45:43.000000000 +0200
+++ control-center-2.4.0/capplets/default-applications/gnome-default-applications-properties.c	2003-09-11 23:20:24.000000000 +0200
@@ -259,7 +259,7 @@
 		if (strcmp (mime_app->command, li_app->command) == 0 &&
 		    mime_app->requires_terminal == li_app->requires_terminal) {
 			gtk_entry_set_text (GTK_ENTRY (WID ("text_select_combo_entry")), _(mime_app->name));
-			gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("text_custom_radio")), TRUE);
+			gtk_widget_set_sensitive(WID ("text_select_combo") ,TRUE);
 			gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("text_select_radio")), TRUE);
 			gnome_vfs_mime_application_free (mime_app);
 			return;
@@ -279,8 +279,9 @@
 	gnome_vfs_mime_set_default_application ("text/plain", mime_app->id);
 	gnome_vfs_application_registry_sync ();	
 
+	
+	
  read_editor_custom:
-	gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("text_select_radio")), TRUE);
 	gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("text_custom_radio")), TRUE);
 	if (mime_app)
 		gnome_vfs_mime_application_free (mime_app);
@@ -388,19 +389,20 @@
 	for (i = 0; i < G_N_ELEMENTS (possible_browsers); i++ ) {
 		if (possible_browsers[i].in_path == FALSE)
 			continue;
-		
-		if (browser && strcmp (browser, possible_browsers[i].command) == 0 &&
-		    needs_term == possible_browsers[i].needs_term) {
+
+		if (browser && (strcmp (browser, possible_browsers[i].command) == 0)
+		&&   needs_term == possible_browsers[i].needs_term) {
 			gtk_entry_set_text (GTK_ENTRY (WID ("web_select_combo_entry")),
 					    _(possible_browsers[i].name));
-			gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("web_custom_radio")), TRUE);
+					    
+			gtk_widget_set_sensitive (WID ("web_select_combo"), TRUE);
 			gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("web_select_radio")), TRUE);
 			g_free (browser);
 			return;
 		}
         }
-	gtk_toggle_button_set_inconsistent (GTK_TOGGLE_BUTTON (WID ("web_select_radio")), TRUE);
-	gtk_toggle_button_set_inconsistent (GTK_TOGGLE_BUTTON (WID ("web_custom_radio")), TRUE);
+	gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("web_custom_radio")), TRUE);
+
 	g_free (browser);
 }
 
@@ -427,18 +429,19 @@
 		if (possible_mailers[i].in_path == FALSE)
 			continue;
 		
-		if (mailer && strcmp (mailer, possible_mailers[i].command) == 0 &&
-		    needs_term == possible_mailers[i].needs_term) {
+		if (mailer && (strcmp (mailer, possible_mailers[i].command) == 0)
+		&&   needs_term == possible_mailers[i].needs_term) {
 			gtk_entry_set_text (GTK_ENTRY (WID ("mail_select_combo_entry")),
 					    _(possible_mailers[i].name));
-			gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("mail_custom_radio")), TRUE);
+					    
+			gtk_widget_set_sensitive (WID ("mail_select_combo"), TRUE);
 			gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("mail_select_radio")), TRUE);
 			g_free (mailer);
 			return;
 		}
         }
-	gtk_toggle_button_set_inconsistent (GTK_TOGGLE_BUTTON (WID ("mail_select_radio")), TRUE);
-	gtk_toggle_button_set_inconsistent (GTK_TOGGLE_BUTTON (WID ("mail_custom_radio")), TRUE);
+	gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("mail_custom_radio")), TRUE);
+
 	g_free (mailer);
 
 	
@@ -510,17 +513,18 @@
 		    strcmp (exec_arg?exec_arg:"", possible_terminals[i].exec_arg) == 0) {
 			gtk_entry_set_text (GTK_ENTRY (WID ("terminal_select_combo_entry")),
 					    _(possible_terminals[i].name));
-			gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("terminal_custom_radio")), TRUE);
+					    
+			gtk_widget_set_sensitive (WID ("terminal_select_combo"), TRUE);
 			gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("terminal_select_radio")), TRUE);
 			g_free (exec);
 			g_free (exec_arg);
 			return;
 		}
         }
-	gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("terminal_select_radio")), TRUE);
 	gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (WID ("terminal_custom_radio")), TRUE);
 	g_free (exec);
 	g_free (exec_arg);
+
 }
 
 static void
