--- control-center-1.4.0.4/configure.in.orig	Sun Feb 10 22:07:34 2002
+++ control-center-1.4.0.4/configure.in	Tue Feb 12 04:20:15 2002
@@ -40,9 +40,6 @@
 CFLAGS=$CFLAGS_save
 LIBS=$LIBS_save
 
-dnl utility conditional
-AM_CONDITIONAL(FALSE, test "x" = "y")
-
 ALL_LINGUAS="az bg ca cs da de el en_GB es et eu fi fr ga gl hr hu it ja ko lt nl nn no pl pt pt_BR ro ru sk sl sv tr uk wa zh_CN zh_TW"
 AM_GNOME_GETTEXT
 
