diff -urN control-center-2.17.92/capplets/keybindings/gnome-keybinding-properties.c control-center-2.17.92.new/capplets/keybindings/gnome-keybinding-properties.c
--- control-center-2.17.92/capplets/keybindings/gnome-keybinding-properties.c	2007-02-26 20:39:50.000000000 +0100
+++ control-center-2.17.92.new/capplets/keybindings/gnome-keybinding-properties.c	2007-03-03 11:13:11.000000000 +0100
@@ -77,6 +77,7 @@
   { "/apps/metacity/window_keybindings/begin_resize",              ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/window_keybindings/toggle_on_all_workspaces",  N_WORKSPACES_GT, 1 },
   /* These next three are out of order compared to the schemas file. */
+  { "/apps/metacity/window_keybindings/toggle_above",              ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/window_keybindings/raise_or_lower",            ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/window_keybindings/raise",                     ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/window_keybindings/lower",                     ALWAYS_VISIBLE,  0 },
@@ -99,10 +100,13 @@
   { "/apps/metacity/window_keybindings/move_to_workspace_up",      N_WORKSPACES_GT, 1 },
   { "/apps/metacity/window_keybindings/move_to_workspace_down",    N_WORKSPACES_GT, 1 },
   { "/apps/metacity/global_keybindings/switch_windows",            ALWAYS_VISIBLE,  0 },
+  { "/apps/metacity/global_keybindings/switch_windows_backwards",  ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/global_keybindings/switch_panels",             ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/global_keybindings/cycle_windows",             ALWAYS_VISIBLE,  0 },
+  { "/apps/metacity/global_keybindings/cycle_windows_backward",    ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/global_keybindings/cycle_group",               ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/global_keybindings/cycle_panels",              ALWAYS_VISIBLE,  0 },
+  { "/apps/metacity/global_keybindings/cycle_panels_backward",     ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/global_keybindings/show_desktop",              ALWAYS_VISIBLE,  0 },
   { "/apps/metacity/global_keybindings/switch_to_workspace_1",     N_WORKSPACES_GT, 1 },
   { "/apps/metacity/global_keybindings/switch_to_workspace_2",     N_WORKSPACES_GT, 1 },
