diff -Nuard control-center-2.4.0.orig/capplets/windows/gnome-window-properties.c control-center-2.4.0/capplets/windows/gnome-window-properties.c
--- control-center-2.4.0.orig/capplets/windows/gnome-window-properties.c	2003-05-01 07:32:13.000000000 +0200
+++ control-center-2.4.0/capplets/windows/gnome-window-properties.c	2003-12-04 23:09:55.000000000 +0100
@@ -47,6 +47,7 @@
 static GnomeWindowManager *current_wm; /* may be NULL */
 static GtkWidget *dialog_win;
 static GtkWidget *focus_mode_checkbutton;
+static GtkWidget *reduced_mode_checkbutton;
 static GtkWidget *autoraise_checkbutton;
 static GtkWidget *autoraise_delay_slider;
 static GtkWidget *autoraise_delay_hbox;
@@ -62,6 +63,11 @@
 static int n_mouse_modifiers = 0;
 
 static void reload_mouse_modifiers (void);
+reduced_mode_toggled_callback (GtkWidget *button, void      *data)
+{
+	gconf_client_set_bool (gconf_client, "/apps/metacity/general/reduced_resources",
+		gtk_toggle_button_get_active (GTK_TOGGLE_BUTTON (button)), NULL);
+}
 
 static void
 mouse_focus_toggled_callback (GtkWidget *button,
@@ -278,6 +284,10 @@
 
         init_settings_struct (&new_settings);
         
+
+        gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (reduced_mode_checkbutton),
+		gconf_client_get_bool (gconf_client, "/apps/metacity/general/reduced_resources",NULL));
+
         if (new_settings.focus_follows_mouse != settings->focus_follows_mouse)
                 gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (focus_mode_checkbutton),
                                               new_settings.focus_follows_mouse);
@@ -452,6 +462,7 @@
         }
         
         dialog_win = WID ("main-dialog");
+        reduced_mode_checkbutton = WID ("reduced-mode-checkbutton");
         focus_mode_checkbutton = WID ("focus-mode-checkbutton");
         autoraise_checkbutton = WID ("autoraise-checkbutton");
         autoraise_delay_slider = WID ("autoraise-delay-slider");
@@ -494,6 +505,8 @@
         g_signal_connect (G_OBJECT (dialog_win), "destroy",
                           G_CALLBACK (gtk_main_quit), NULL);
 
+        g_signal_connect (G_OBJECT (reduced_mode_checkbutton), "toggled",
+                          G_CALLBACK (reduced_mode_toggled_callback), NULL);
 
         g_signal_connect (G_OBJECT (focus_mode_checkbutton), "toggled",
                           G_CALLBACK (mouse_focus_toggled_callback), NULL);
diff -Nuard control-center-2.4.0.orig/capplets/windows/gnome-window-properties.glade control-center-2.4.0/capplets/windows/gnome-window-properties.glade
--- control-center-2.4.0.orig/capplets/windows/gnome-window-properties.glade	2003-08-04 17:33:23.000000000 +0200
+++ control-center-2.4.0/capplets/windows/gnome-window-properties.glade	2003-12-04 21:06:38.000000000 +0100
@@ -12,6 +12,11 @@
   <property name="modal">False</property>
   <property name="resizable">False</property>
   <property name="destroy_with_parent">False</property>
+  <property name="decorated">True</property>
+  <property name="skip_taskbar_hint">False</property>
+  <property name="skip_pager_hint">False</property>
+  <property name="type_hint">GDK_WINDOW_TYPE_HINT_DIALOG</property>
+  <property name="gravity">GDK_GRAVITY_NORTH_WEST</property>
   <property name="has_separator">False</property>
 
   <child internal-child="vbox">
@@ -33,6 +38,7 @@
 	      <property name="label">gtk-help</property>
 	      <property name="use_stock">True</property>
 	      <property name="relief">GTK_RELIEF_NORMAL</property>
+	      <property name="focus_on_click">True</property>
 	      <property name="response_id">-11</property>
 	    </widget>
 	  </child>
@@ -46,6 +52,7 @@
 	      <property name="label">gtk-close</property>
 	      <property name="use_stock">True</property>
 	      <property name="relief">GTK_RELIEF_NORMAL</property>
+	      <property name="focus_on_click">True</property>
 	      <property name="response_id">-7</property>
 	    </widget>
 	  </child>
@@ -111,6 +118,7 @@
 			  <property name="label" translatable="yes">_Select windows when the mouse moves over them</property>
 			  <property name="use_underline">True</property>
 			  <property name="relief">GTK_RELIEF_NORMAL</property>
+			  <property name="focus_on_click">True</property>
 			  <property name="active">False</property>
 			  <property name="inconsistent">False</property>
 			  <property name="draw_indicator">True</property>
@@ -141,6 +149,7 @@
 				  <property name="label" translatable="yes">_Raise selected windows after an interval</property>
 				  <property name="use_underline">True</property>
 				  <property name="relief">GTK_RELIEF_NORMAL</property>
+				  <property name="focus_on_click">True</property>
 				  <property name="active">False</property>
 				  <property name="inconsistent">False</property>
 				  <property name="draw_indicator">True</property>
@@ -200,8 +209,8 @@
 					  <property name="inverted">False</property>
 					  <property name="adjustment">7.7 0 10 0.2 1 0</property>
 					  <accessibility>
-						<atkrelation target="autoraise-delay-label" type="labelled-by"/>
-						<atkrelation target="label1" type="labelled-by"/>
+					    <atkrelation target="label1" type="labelled-by"/>
+					    <atkrelation target="autoraise-delay-label" type="labelled-by"/>
 					  </accessibility>
 					</widget>
 					<packing>
@@ -225,7 +234,7 @@
 					  <property name="xpad">4</property>
 					  <property name="ypad">0</property>
 					  <accessibility>
-						<atkrelation target="autoraise-delay-slider" type="label-for"/>
+					    <atkrelation target="autoraise-delay-slider" type="label-for"/>
 					  </accessibility>
 					</widget>
 					<packing>
@@ -285,6 +294,85 @@
 	  </child>
 
 	  <child>
+	    <widget class="GtkVBox" id="vbox9">
+	      <property name="visible">True</property>
+	      <property name="homogeneous">False</property>
+	      <property name="spacing">12</property>
+
+	      <child>
+		<widget class="GtkLabel" id="label6">
+		  <property name="visible">True</property>
+		  <property name="label" translatable="yes">&lt;b&gt;Reduced Resources&lt;/b&gt;</property>
+		  <property name="use_underline">False</property>
+		  <property name="use_markup">True</property>
+		  <property name="justify">GTK_JUSTIFY_LEFT</property>
+		  <property name="wrap">False</property>
+		  <property name="selectable">False</property>
+		  <property name="xalign">0</property>
+		  <property name="yalign">0.5</property>
+		  <property name="xpad">0</property>
+		  <property name="ypad">0</property>
+		</widget>
+		<packing>
+		  <property name="padding">0</property>
+		  <property name="expand">False</property>
+		  <property name="fill">False</property>
+		</packing>
+	      </child>
+
+	      <child>
+		<widget class="GtkHBox" id="hbox16">
+		  <property name="visible">True</property>
+		  <property name="homogeneous">False</property>
+		  <property name="spacing">0</property>
+
+		  <child>
+		    <widget class="GtkVBox" id="vbox10">
+		      <property name="visible">True</property>
+		      <property name="homogeneous">False</property>
+		      <property name="spacing">12</property>
+
+		      <child>
+			<widget class="GtkCheckButton" id="reduced-mode-checkbutton">
+			  <property name="visible">True</property>
+			  <property name="can_focus">True</property>
+			  <property name="label" translatable="yes">_Use wireframe move/resize</property>
+			  <property name="use_underline">True</property>
+			  <property name="relief">GTK_RELIEF_NORMAL</property>
+			  <property name="focus_on_click">True</property>
+			  <property name="active">False</property>
+			  <property name="inconsistent">False</property>
+			  <property name="draw_indicator">True</property>
+			</widget>
+			<packing>
+			  <property name="padding">0</property>
+			  <property name="expand">False</property>
+			  <property name="fill">False</property>
+			</packing>
+		      </child>
+		    </widget>
+		    <packing>
+		      <property name="padding">18</property>
+		      <property name="expand">True</property>
+		      <property name="fill">True</property>
+		    </packing>
+		  </child>
+		</widget>
+		<packing>
+		  <property name="padding">0</property>
+		  <property name="expand">True</property>
+		  <property name="fill">True</property>
+		</packing>
+	      </child>
+	    </widget>
+	    <packing>
+	      <property name="padding">0</property>
+	      <property name="expand">True</property>
+	      <property name="fill">True</property>
+	    </packing>
+	  </child>
+
+	  <child>
 	    <widget class="GtkVBox" id="vbox5">
 	      <property name="visible">True</property>
 	      <property name="homogeneous">False</property>
diff -Nuard control-center-2.4.0.orig/po/control-center-2.0.pot control-center-2.4.0/po/control-center-2.0.pot
--- control-center-2.4.0.orig/po/control-center-2.0.pot	2003-08-27 21:02:41.000000000 +0200
+++ control-center-2.4.0/po/control-center-2.0.pot	2003-12-05 00:31:19.744571640 +0100
@@ -2305,6 +2305,14 @@
 msgid "_Select windows when the mouse moves over them"
 msgstr ""
 
+#: capplets/windows/gnome-window-properties.glade.h:7
+msgid "<b>Reduced Resources</b>"
+msgstr ""
+
+#: capplets/windows/gnome-window-properties.glade.h:7
+msgid "_Use wireframe move/resize"
+msgstr ""
+
 #: capplets/windows/window-properties.desktop.in.in.h:1
 msgid "Set your window properties"
 msgstr ""
diff -Nuard control-center-2.4.0.orig/po/pl.po control-center-2.4.0/po/pl.po
--- control-center-2.4.0.orig/po/pl.po	2003-09-02 21:34:47.000000000 +0200
+++ control-center-2.4.0/po/pl.po	2003-12-05 00:33:08.050106696 +0100
@@ -2420,6 +2420,14 @@
 msgid "_Select windows when the mouse moves over them"
 msgstr "_Zaznaczanie okna po umieszczeniu wskaźnika nad jego obszarem"
 
+#: capplets/windows/gnome-window-properties.glade.h:7
+msgid "<b>Reduced Resources</b>"
+msgstr "<b>Zredukowane zasoby</b>"
+
+#: capplets/windows/gnome-window-properties.glade.h:7
+msgid "_Use wireframe move/resize"
+msgstr "_Ramka podczas przenoszenia/skalowania okna"
+
 #: capplets/windows/window-properties.desktop.in.in.h:1
 msgid "Set your window properties"
 msgstr "Ustawienia okien"
