diff -ur control-center-1.4.0.5.orig/capplets/background-properties/background-properties.desktop.in control-center-1.4.0.5/capplets/background-properties/background-properties.desktop.in
--- control-center-1.4.0.5.orig/capplets/background-properties/background-properties.desktop.in	Fri Oct 26 10:59:59 2001
+++ control-center-1.4.0.5/capplets/background-properties/background-properties.desktop.in	Fri Dec 19 14:43:34 2003
@@ -3,5 +3,7 @@
 _Comment=Configuration of the desktop's background
 Exec=background-properties-capplet
 Icon=gnome-ccbackground.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/default-applications/default-application-properties.desktop.in control-center-1.4.0.5/capplets/default-applications/default-application-properties.desktop.in
--- control-center-1.4.0.5.orig/capplets/default-applications/default-application-properties.desktop.in	Sat Apr  6 15:46:16 2002
+++ control-center-1.4.0.5/capplets/default-applications/default-application-properties.desktop.in	Fri Dec 19 14:42:36 2003
@@ -3,5 +3,7 @@
 _Comment=Choose the applications used by default
 Exec=default-application-properties-capplet
 Icon=
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/file-types/file-types-capplet.desktop.in control-center-1.4.0.5/capplets/file-types/file-types-capplet.desktop.in
--- control-center-1.4.0.5.orig/capplets/file-types/file-types-capplet.desktop.in	Fri Oct 26 11:00:07 2001
+++ control-center-1.4.0.5/capplets/file-types/file-types-capplet.desktop.in	Fri Dec 19 14:43:27 2003
@@ -3,5 +3,7 @@
 _Comment=Specify which programs are used to open or view each file type
 Icon=gnome-ccmime.png
 Exec=file-types-capplet
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/keyboard-properties/keyboard-properties.desktop.in control-center-1.4.0.5/capplets/keyboard-properties/keyboard-properties.desktop.in
--- control-center-1.4.0.5.orig/capplets/keyboard-properties/keyboard-properties.desktop.in	Fri Oct 26 10:59:59 2001
+++ control-center-1.4.0.5/capplets/keyboard-properties/keyboard-properties.desktop.in	Fri Dec 19 14:34:39 2003
@@ -3,5 +3,7 @@
 _Comment=Keyboard Properties
 Exec=keyboard-properties
 Icon=gnome-cckeyboard.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/mouse-properties/mouse-properties.desktop.in control-center-1.4.0.5/capplets/mouse-properties/mouse-properties.desktop.in
--- control-center-1.4.0.5.orig/capplets/mouse-properties/mouse-properties.desktop.in	Fri Oct 26 11:00:00 2001
+++ control-center-1.4.0.5/capplets/mouse-properties/mouse-properties.desktop.in	Fri Dec 19 14:43:21 2003
@@ -3,5 +3,7 @@
 _Comment=Mouse Properties
 Exec=mouse-properties-capplet
 Icon=gnome-mouse.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/sound-properties/sound-properties.desktop.in control-center-1.4.0.5/capplets/sound-properties/sound-properties.desktop.in
--- control-center-1.4.0.5.orig/capplets/sound-properties/sound-properties.desktop.in	Fri Oct 26 11:00:05 2001
+++ control-center-1.4.0.5/capplets/sound-properties/sound-properties.desktop.in	Fri Dec 19 14:34:32 2003
@@ -3,5 +3,7 @@
 _Comment=Configure GNOME's use of sound
 Exec=sound-properties
 Icon=gnome-audio2.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/theme-switcher/theme-selector.desktop.in control-center-1.4.0.5/capplets/theme-switcher/theme-selector.desktop.in
--- control-center-1.4.0.5.orig/capplets/theme-switcher/theme-selector.desktop.in	Fri Oct 26 11:00:05 2001
+++ control-center-1.4.0.5/capplets/theme-switcher/theme-selector.desktop.in	Fri Dec 19 14:43:14 2003
@@ -3,5 +3,7 @@
 _Comment=Select which desktop theme to use
 Exec=theme-selector-capplet
 Icon=gnome-ccthemes.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/ui-properties/application.desktop.in control-center-1.4.0.5/capplets/ui-properties/application.desktop.in
--- control-center-1.4.0.5.orig/capplets/ui-properties/application.desktop.in	Fri Oct 26 11:00:06 2001
+++ control-center-1.4.0.5/capplets/ui-properties/application.desktop.in	Fri Dec 19 14:34:24 2003
@@ -3,5 +3,7 @@
 _Comment=Sets the default feel of GNOME applications
 Exec=ui-properties --cap-id=1
 Icon=gnome-applications.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/ui-properties/dialogs.desktop.in control-center-1.4.0.5/capplets/ui-properties/dialogs.desktop.in
--- control-center-1.4.0.5.orig/capplets/ui-properties/dialogs.desktop.in	Fri Oct 26 11:00:06 2001
+++ control-center-1.4.0.5/capplets/ui-properties/dialogs.desktop.in	Fri Dec 19 14:43:07 2003
@@ -2,5 +2,7 @@
 _Name=Dialogs
 Exec=ui-properties --cap-id=0
 Icon=gnome-ccdialog.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/ui-properties/mdi.desktop.in control-center-1.4.0.5/capplets/ui-properties/mdi.desktop.in
--- control-center-1.4.0.5.orig/capplets/ui-properties/mdi.desktop.in	Fri Oct 26 11:00:06 2001
+++ control-center-1.4.0.5/capplets/ui-properties/mdi.desktop.in	Fri Dec 19 14:34:16 2003
@@ -2,5 +2,7 @@
 _Name=MDI
 Exec=ui-properties --cap-id=2
 Icon=gnome-mdi.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/url-properties/url-properties.desktop.in control-center-1.4.0.5/capplets/url-properties/url-properties.desktop.in
--- control-center-1.4.0.5.orig/capplets/url-properties/url-properties.desktop.in	Fri Oct 26 11:00:06 2001
+++ control-center-1.4.0.5/capplets/url-properties/url-properties.desktop.in	Fri Dec 19 14:43:01 2003
@@ -3,5 +3,7 @@
 _Comment=Configure which programs are used to display URLs
 Exec=url-properties
 Icon=gnome-html.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/capplets/wm-properties/wm-properties.desktop.in control-center-1.4.0.5/capplets/wm-properties/wm-properties.desktop.in
--- control-center-1.4.0.5.orig/capplets/wm-properties/wm-properties.desktop.in	Fri Oct 26 11:00:06 2001
+++ control-center-1.4.0.5/capplets/wm-properties/wm-properties.desktop.in	Fri Dec 19 14:43:46 2003
@@ -3,5 +3,7 @@
 _Comment=Choose a window manager
 Exec=wm-properties-capplet --sync
 Icon=gnome-ccwindowmanager.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
diff -ur control-center-1.4.0.5.orig/control-center/gnomecc.desktop.in control-center-1.4.0.5/control-center/gnomecc.desktop.in
--- control-center-1.4.0.5.orig/control-center/gnomecc.desktop.in	Fri Oct 26 11:00:08 2001
+++ control-center-1.4.0.5/control-center/gnomecc.desktop.in	Fri Dec 19 14:42:43 2003
@@ -3,5 +3,7 @@
 _Comment=The GNOME configuration tool
 Icon=control-center.png
 Exec=gnomecc
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
--- control-center-1.4.0.5/acinclude.m4~	Fri Oct 26 11:00:07 2001
+++ control-center-1.4.0.5/acinclude.m4	Fri Dec 19 16:21:28 2003
@@ -1,3 +1,138 @@
+## xml-i18n-tools.m4 - Configure xml-i18n-tools for the target system. -*-Shell-script-*-
+## Copyright (C) 2001 Eazel, Inc.
+## Author: Maciej Stachowiak <mjs@noisehavoc.org>
+##
+## This program is free software; you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation; either version 2 of the License, or
+## (at your option) any later version.
+##
+## This program is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with this program; if not, write to the Free Software
+## Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+##
+## As a special exception to the GNU General Public License, if you
+## distribute this file as part of a program that contains a
+## configuration script generated by Autoconf, you may include it under
+## the same distribution terms that you use for the rest of that program.
+
+AC_DEFUN(XML_I18N_TOOLS_NEWER_THAN_0_9,[ true ])
+
+dnl AC_PROG_XML_I18N_TOOLS([MINIMUM-VERSION [,"G2" if always using --utf8] ])
+# serial 1 AC_PROG_XML_I18N_TOOLS
+AC_DEFUN(AC_PROG_XML_I18N_TOOLS,
+[
+  AC_DEFUN(X18T_PFORG1,  dnl and -u for G2
+        ifelse([$2],[G2],[ -u ], [ -p ]))
+  AC_DEFUN(X18T_XML_KIND, 
+        ifelse([$2],[G2],[ -u ],[ $(XML_I18N_XML_KIND) ]))
+  AC_DEFUN(X18T_KEYS_KIND, 
+        ifelse([$2],[G2],[ -u ],[ $(XML_I18N_KEYS_KIND) ]))
+
+  XML_I18N_MERGE_DESKTOP_RULE='%.desktop:   %.desktop.in   $(top_builddir)/xml-i18n-merge $(wildcard $(top_srcdir)/po/*.po) ; $(top_builddir)/xml-i18n-merge $(top_srcdir)/po $< [$]@ -d -u X18T_PFORG1'
+XML_I18N_MERGE_DIRECTORY_RULE='%.directory: %.directory.in $(top_builddir)/xml-i18n-merge $(wildcard $(top_srcdir)/po/*.po) ; $(top_builddir)/xml-i18n-merge $(top_srcdir)/po $< [$]@ -d -u X18T_PFORG1'
+     XML_I18N_MERGE_KEYS_RULE='%.keys:      %.keys.in      $(top_builddir)/xml-i18n-merge $(wildcard $(top_srcdir)/po/*.po) ; $(top_builddir)/xml-i18n-merge $(top_srcdir)/po $< [$]@ -k X18T_KEYS_KIND'
+      XML_I18N_MERGE_OAF_RULE='%.oaf:       %.oaf.in       $(top_builddir)/xml-i18n-merge $(wildcard $(top_srcdir)/po/*.po) ; $(top_builddir)/xml-i18n-merge $(top_srcdir)/po $< [$]@ -o -p'
+     XML_I18N_MERGE_PONG_RULE='%.pong:      %.pong.in      $(top_builddir)/xml-i18n-merge $(wildcard $(top_srcdir)/po/*.po) ; $(top_builddir)/xml-i18n-merge $(top_srcdir)/po $< [$]@ -x X18T_PFORG1'
+   XML_I18N_MERGE_SERVER_RULE='%.server:    %.server.in    $(top_builddir)/xml-i18n-merge $(wildcard $(top_srcdir)/po/*.po) ; $(top_builddir)/xml-i18n-merge $(top_srcdir)/po $< [$]@ -o -u'
+    XML_I18N_MERGE_SHEET_RULE='%.sheet:     %.sheet.in     $(top_builddir)/xml-i18n-merge $(wildcard $(top_srcdir)/po/*.po) ; $(top_builddir)/xml-i18n-merge $(top_srcdir)/po $< [$]@ -x -u'
+XML_I18N_MERGE_SOUNDLIST_RULE='%.soundlist: %.soundlist.in $(top_builddir)/xml-i18n-merge $(wildcard $(top_srcdir)/po/*.po) ; $(top_builddir)/xml-i18n-merge $(top_srcdir)/po $< [$]@ -d X18T_PFORG1'
+      XML_I18N_MERGE_XML_RULE='%.xml:       %.xml.in       $(top_builddir)/xml-i18n-merge $(wildcard $(top_srcdir)/po/*.po) ; $(top_builddir)/xml-i18n-merge $(top_srcdir)/po $< [$]@ -x X18T_XML_KIND'
+
+AC_SUBST(XML_I18N_MERGE_DESKTOP_RULE)
+AC_SUBST(XML_I18N_MERGE_DIRECTORY_RULE)
+AC_SUBST(XML_I18N_MERGE_KEYS_RULE)
+AC_SUBST(XML_I18N_MERGE_OAF_RULE)
+AC_SUBST(XML_I18N_MERGE_PONG_RULE)
+AC_SUBST(XML_I18N_MERGE_SERVER_RULE)
+AC_SUBST(XML_I18N_MERGE_SHEET_RULE)
+AC_SUBST(XML_I18N_MERGE_SOUNDLIST_RULE)
+AC_SUBST(XML_I18N_MERGE_XML_RULE)
+
+# Use the tools built into the package, not the ones that are installed.
+
+XML_I18N_EXTRACT='$(top_builddir)/xml-i18n-extract'
+AC_SUBST(XML_I18N_EXTRACT)dnl
+
+XML_I18N_MERGE='$(top_builddir)/xml-i18n-merge'
+AC_SUBST(XML_I18N_MERGE)dnl
+
+XML_I18N_UPDATE='$(top_builddir)/xml-i18n-update'
+AC_SUBST(XML_I18N_UPDATE)dnl
+
+AC_PATH_PROG(INTLTOOL_PERL, perl)
+if test -z "$INTLTOOL_PERL"; then
+   AC_MSG_ERROR([perl not found; required for xml-i18n-tools])
+fi
+if test -z "`$INTLTOOL_PERL -v | fgrep '5.' 2> /dev/null`"; then
+   AC_MSG_ERROR([perl 5.x required for xml-i18n-tools])
+fi
+
+dnl Remove Intltool [] tags from po/POTFILES
+dnl
+ifdef([AC_DIVERSION_ICMDS],[
+  AC_DIVERT_PUSH(AC_DIVERSION_ICMDS)
+      changequote(,)
+      mv -f po/POTFILES po/POTFILES.tmp
+      sed -e 's/\[.*\] *//' < po/POTFILES.tmp > po/POTFILES
+      rm -f po/POTFILES.tmp
+      changequote([,])
+  AC_DIVERT_POP()
+],[
+  ifdef([AC_CONFIG_COMMANDS_PRE],[
+    AC_CONFIG_COMMANDS_PRE([
+        changequote(,)
+        mv -f po/POTFILES po/POTFILES.tmp
+        sed -e 's/\[.*\] *//' < po/POTFILES.tmp > po/POTFILES
+        rm -f po/POTFILES.tmp
+        changequote([,])
+    ])
+  ])
+])
+
+dnl  manually sed perl in so people don't have to put the xml-i18n-tools scripts in their 
+dnl  AC_OUTPUT
+AC_OUTPUT_COMMANDS([
+sed -e "s:@INTLTOOL_PERL@:${INTLTOOL_PERL}:;" < ${srcdir}/xml-i18n-extract.in > xml-i18n-extract.out
+if cmp -s xml-i18n-extract xml-i18n-extract.out 2>/dev/null; then
+  rm -f xml-i18n-extract.out
+else
+  mv -f xml-i18n-extract.out xml-i18n-extract
+fi
+chmod ugo+x xml-i18n-extract
+chmod u+w xml-i18n-extract
+
+sed -e "s:@INTLTOOL_PERL@:${INTLTOOL_PERL}:;" < ${srcdir}/xml-i18n-merge.in > xml-i18n-merge.out
+if cmp -s xml-i18n-merge xml-i18n-merge.out 2>/dev/null; then
+  rm -f xml-i18n-merge.out
+else
+  mv -f xml-i18n-merge.out xml-i18n-merge
+fi
+chmod ugo+x xml-i18n-merge
+chmod u+w xml-i18n-merge
+
+sed -e "s:@INTLTOOL_PERL@:${INTLTOOL_PERL}:;" < ${srcdir}/xml-i18n-update.in > xml-i18n-update.out
+if cmp -s xml-i18n-update xml-i18n-update.out 2>/dev/null; then
+  rm -f xml-i18n-update.out
+else
+  mv -f xml-i18n-update.out xml-i18n-update
+fi
+chmod ugo+x xml-i18n-update
+chmod u+w xml-i18n-update
+], INTLTOOL_PERL=${INTLTOOL_PERL})
+
+# Redirect the config.log output again, so that the ltconfig log is not
+# clobbered by the next message.
+exec 5>>./config.log
+])
+
+dnl old names
+AC_DEFUN(AM_PROG_XML_I18N_TOOLS, [indir([AC_PROG_XML_I18N_TOOLS])])dnl
 # Configure paths for ESD
 # Manish Singh    98-9-30
 # stolen back from Frank Belew
