diff -Nuard control-center-2.1.2.orig/capplets/mouse/Makefile.am control-center-2.1.2/capplets/mouse/Makefile.am
--- control-center-2.1.2.orig/capplets/mouse/Makefile.am	Sat Nov  2 19:40:22 2002
+++ control-center-2.1.2/capplets/mouse/Makefile.am	Tue Nov 12 16:16:22 2002
@@ -44,7 +44,7 @@
 	mouse_dblclck_off_data $(srcdir)/double-click-off.png
 
 BUILT_SOURCES = inlinepixbufs.h
-CLEANFILES = inlinepixbufs.h
+#CLEANFILES = inlinepixbufs.h
 
 inlinepixbufs.h: $(IMAGES)
 	$(GDK_PIXBUF_CSOURCE) --raw --build-list $(VARIABLES) >$(srcdir)/inlinepixbufs.h
