diff -urN control-center-2.6.1.orig/configure.in control-center-2.6.1/configure.in
--- control-center-2.6.1.orig/configure.in	2004-04-16 11:19:29.267910848 +0200
+++ control-center-2.6.1/configure.in	2004-04-16 11:19:41.157103416 +0200
@@ -32,7 +32,7 @@
 changequote([,])dnl
 
 # Internationalization support
-ALL_LINGUAS="af am ar az be bg bn bs ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gl gu he hi hr hu id is it ja ko lt lv mk ml mn ms nb nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="af am ar az be bg bn bs ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gl gu he hi hr hu id is it ja ko lt lv mk ml mn ms nb nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi wa zh_CN zh_TW"
 
 GETTEXT_PACKAGE=control-center-2.0
 AC_SUBST(GETTEXT_PACKAGE)
