diff -urN control-center-2.8.2.orig/capplets/default-applications/gnome-default-applications-properties-structs.c control-center-2.8.2/capplets/default-applications/gnome-default-applications-properties-structs.c
--- control-center-2.8.2.orig/capplets/default-applications/gnome-default-applications-properties-structs.c	2005-02-20 22:18:03.000000000 +0100
+++ control-center-2.8.2/capplets/default-applications/gnome-default-applications-properties-structs.c	2005-02-25 13:50:29.483554672 +0100
@@ -10,7 +10,7 @@
         { N_("Galeon"), 		"galeon",	"galeon %s",		FALSE, FALSE },
         { N_("Encompass"), 		"encompass",	"encompass %s",		FALSE, FALSE },
         { N_("Firebird/FireFox"),  "mozilla-firebird",  "mozilla-firebird %s",  FALSE, FALSE },
-        { N_("Firebird/FireFox"),  "mozilla-firefox",	"mozilla-firebird %s",  FALSE, FALSE },
+        { N_("Firebird/FireFox"),  "mozilla-firefox",	"mozilla-firefox %s",  FALSE, FALSE },
         { N_("Firebird/FireFox"),  	"firefox",	"mozilla-firefox %s",	FALSE, FALSE },
         { N_("Mozilla/Netscape 6"), 	"mozilla-1.6",	"mozilla-1.6 %s",	FALSE, FALSE },
         { N_("Mozilla/Netscape 6"), 	"mozilla",	"mozilla %s",		FALSE, FALSE },
