diff -aurN control-center-2.13.90.orig/control-center/gnomecc.desktop.in.in control-center-2.13.90/control-center/gnomecc.desktop.in.in
--- control-center-2.13.90.orig/control-center/gnomecc.desktop.in.in	2006-01-23 12:36:20.000000000 +0100
+++ control-center-2.13.90/control-center/gnomecc.desktop.in.in	2006-01-30 22:53:02.990754000 +0100
@@ -8,7 +8,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Application;Settings;
+Categories=GTK;GNOME;Settings;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
