--- control-center-1.4.0.1/capplets/wm-properties/wm-list.c~	Wed Aug  4 03:27:15 1999
+++ control-center-1.4.0.1/capplets/wm-properties/wm-list.c	Fri Apr 27 13:12:41 2001
@@ -228,7 +228,7 @@
         gchar *tempdir;
         gchar *name;
         
-        tempdir = gnome_unconditional_datadir_file ("gnome/wm-properties/");
+        tempdir = gnome_unconditional_datadir_file ("wm-properties/");
         wm_list_read_dir (tempdir, FALSE);
         g_free (tempdir);
         
