diff -aurN control-center-2.15.91.orig/gnome-settings-daemon/gnome-settings-dbus.c control-center-2.15.91/gnome-settings-daemon/gnome-settings-dbus.c
--- control-center-2.15.91.orig/gnome-settings-daemon/gnome-settings-dbus.c	2006-07-06 21:44:01.000000000 +0200
+++ control-center-2.15.91/gnome-settings-daemon/gnome-settings-dbus.c	2006-08-09 18:55:58.680735010 +0200
@@ -1,4 +1,4 @@
-#include <dbus/dbus-glib-bindings.h>
+#include <dbus/dbus-glib.h>
 #include <gnome-settings-daemon.h>
 
 static GObjectClass *parent_class = NULL;
@@ -75,7 +75,7 @@
 						  DBUS_PATH_DBUS,
 						  DBUS_INTERFACE_DBUS);
 
-	if (!org_freedesktop_DBus_request_name
+	if (!dbus_bus_request_name
 	    (driver_proxy, "org.gnome.SettingsDaemon", 0, &request_ret,
 	     &error)) {
 		g_warning ("Unable to register service: %s",
diff -aurN control-center-2.15.91.orig/libgswitchit/keyboard-config-registry.c control-center-2.15.91/libgswitchit/keyboard-config-registry.c
--- control-center-2.15.91.orig/libgswitchit/keyboard-config-registry.c	2006-08-06 23:46:42.000000000 +0200
+++ control-center-2.15.91/libgswitchit/keyboard-config-registry.c	2006-08-09 18:56:22.705082754 +0200
@@ -146,7 +146,7 @@
 						  DBUS_PATH_DBUS,
 						  DBUS_INTERFACE_DBUS);
 
-	if (!org_freedesktop_DBus_request_name
+	if (!dbus_bus_request_name
 	    (driver_proxy, "org.gnome.KeyboardConfigRegistry", 0,
 	     &request_ret, &error)) {
 		g_warning ("Unable to register service: %s",
diff -aurN control-center-2.15.91.orig/libgswitchit/keyboard-config-registry.h control-center-2.15.91/libgswitchit/keyboard-config-registry.h
--- control-center-2.15.91.orig/libgswitchit/keyboard-config-registry.h	2006-07-06 21:42:56.000000000 +0200
+++ control-center-2.15.91/libgswitchit/keyboard-config-registry.h	2006-08-09 18:15:13.562449490 +0200
@@ -1,7 +1,7 @@
 #ifndef __KEYBOARD_CONFIG_REGISTRY_H__
 #define __KEYBOARD_CONFIG_REGISTRY_H__
 
-#include <dbus/dbus-glib-bindings.h>
+#include <dbus/dbus-glib.h>
 #include <libxklavier/xklavier.h>
 
 typedef struct KeyboardConfigRegistry KeyboardConfigRegistry;
