diff -urN control-center-1.4.0.1.org/configure.in control-center-1.4.0.1/configure.in
--- control-center-1.4.0.1.org/configure.in	Sat Apr  7 15:26:26 2001
+++ control-center-1.4.0.1/configure.in	Sat Apr  7 15:26:51 2001
@@ -44,7 +44,7 @@
 AM_CONDITIONAL(FALSE, test "x" = "y")
 
 ALL_LINGUAS="az bg ca cs da de el en_GB es et eu fi fr ga gl hr hu it ja ko lt nl no pl pt pt_BR ro ru sk sl sv tr uk wa zh_CN.GB2312 zh_TW.Big5"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 
 AC_SUBST(CFLAGS)
 AC_SUBST(LDFLAGS)
