diff -aurN control-center-2.13.90.orig/vfs-methods/fontilus/Makefile.am control-center-2.13.90/vfs-methods/fontilus/Makefile.am
--- control-center-2.13.90.orig/vfs-methods/fontilus/Makefile.am	2004-12-20 22:53:30.000000000 +0100
+++ control-center-2.13.90/vfs-methods/fontilus/Makefile.am	2006-01-30 23:02:21.060754000 +0100
@@ -50,7 +50,6 @@
 endif
 
 install-desktop-database: install-desktopDATA
-	update-desktop-database $(DESTDIR)$(desktopdir)
 
 uninstall-local:
 	rm -f $(DESTDIR)$(desktopdir)/mimeinfo.cache
diff -aurN control-center-2.13.90.orig/vfs-methods/themus/Makefile.am control-center-2.13.90/vfs-methods/themus/Makefile.am
--- control-center-2.13.90.orig/vfs-methods/themus/Makefile.am	2004-12-20 22:53:31.000000000 +0100
+++ control-center-2.13.90/vfs-methods/themus/Makefile.am	2006-01-30 23:02:21.060754000 +0100
@@ -71,7 +71,6 @@
 endif
 
 install-desktop-database: install-desktopDATA
-	update-desktop-database $(DESTDIR)$(desktopdir)
 
 distclean-local:
 	rm -f $(DESTDIR)$(desktopdir)/mimeinfo.cache
