diff -urN control-center-2.5.4.orig/configure.in control-center-2.5.4/configure.in
--- control-center-2.5.4.orig/configure.in	2004-03-11 10:16:46.582272424 +0100
+++ control-center-2.5.4/configure.in	2004-03-11 10:16:58.697430640 +0100
@@ -32,7 +32,7 @@
 changequote([,])dnl
 
 # Internationalization support
-ALL_LINGUAS="af am ar az be bg bn ca cs cy da de el en_CA en_GB es et fa fi fr ga gl he hi hr hu id is it ja ko lt lv mk ml mn ms nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="af am ar az be bg bn ca cs cy da de el en_CA en_GB es et fa fi fr ga gl he hi hr hu id is it ja ko lt lv mk ml mn ms nb nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi wa zh_CN zh_TW"
 
 GETTEXT_PACKAGE=control-center-2.0
 AC_SUBST(GETTEXT_PACKAGE)
