diff -ur control-center-2.8.0.orig/capplets/accessibility/at-properties/at-properties.desktop.in.in control-center-2.8.0/capplets/accessibility/at-properties/at-properties.desktop.in.in
--- control-center-2.8.0.orig/capplets/accessibility/at-properties/at-properties.desktop.in.in	2003-12-16 22:38:05.000000000 +0100
+++ control-center-2.8.0/capplets/accessibility/at-properties/at-properties.desktop.in.in	2004-09-16 22:40:26.432812736 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Settings;Accessibility;
+Categories=GTK;GNOME;Settings;Accessibility;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/accessibility/keyboard/accessibility-keyboard.desktop.in.in control-center-2.8.0/capplets/accessibility/keyboard/accessibility-keyboard.desktop.in.in
--- control-center-2.8.0.orig/capplets/accessibility/keyboard/accessibility-keyboard.desktop.in.in	2003-12-16 22:38:05.000000000 +0100
+++ control-center-2.8.0/capplets/accessibility/keyboard/accessibility-keyboard.desktop.in.in	2004-09-16 22:40:37.486132376 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;Accessibility;
+Categories=GTK;GNOME;Settings;Accessibility;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/background/background.desktop.in.in control-center-2.8.0/capplets/background/background.desktop.in.in
--- control-center-2.8.0.orig/capplets/background/background.desktop.in.in	2004-02-04 23:04:14.000000000 +0100
+++ control-center-2.8.0/capplets/background/background.desktop.in.in	2004-09-16 22:40:49.479309136 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Settings;Appearance;
+Categories=GTK;GNOME;Settings;Appearance;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/default-applications/default-applications.desktop.in.in control-center-2.8.0/capplets/default-applications/default-applications.desktop.in.in
--- control-center-2.8.0.orig/capplets/default-applications/default-applications.desktop.in.in	2003-12-16 22:38:07.000000000 +0100
+++ control-center-2.8.0/capplets/default-applications/default-applications.desktop.in.in	2004-09-16 22:42:01.076424720 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;AdvancedSettings;
+Categories=GTK;GNOME;Settings;AdvancedSettings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/display/display-properties.desktop.in.in control-center-2.8.0/capplets/display/display-properties.desktop.in.in
--- control-center-2.8.0.orig/capplets/display/display-properties.desktop.in.in	2003-04-23 17:01:16.000000000 +0200
+++ control-center-2.8.0/capplets/display/display-properties.desktop.in.in	2004-09-16 22:42:25.259748296 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Settings;Appearance;
+Categories=GTK;GNOME;Settings;Appearance;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/font/font-properties.desktop.in.in control-center-2.8.0/capplets/font/font-properties.desktop.in.in
--- control-center-2.8.0.orig/capplets/font/font-properties.desktop.in.in	2003-12-16 22:38:11.000000000 +0100
+++ control-center-2.8.0/capplets/font/font-properties.desktop.in.in	2004-09-16 22:47:31.484195104 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Settings;Appearance;
+Categories=GTK;GNOME;Settings;Appearance;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/keybindings/keybinding.desktop.in.in control-center-2.8.0/capplets/keybindings/keybinding.desktop.in.in
--- control-center-2.8.0.orig/capplets/keybindings/keybinding.desktop.in.in	2003-12-16 22:38:13.000000000 +0100
+++ control-center-2.8.0/capplets/keybindings/keybinding.desktop.in.in	2004-09-16 22:47:54.745658824 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/keyboard/keyboard.desktop.in.in control-center-2.8.0/capplets/keyboard/keyboard.desktop.in.in
--- control-center-2.8.0.orig/capplets/keyboard/keyboard.desktop.in.in	2003-12-16 22:38:14.000000000 +0100
+++ control-center-2.8.0/capplets/keyboard/keyboard.desktop.in.in	2004-09-16 22:48:42.817350816 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/mouse/gnome-settings-mouse.desktop.in.in control-center-2.8.0/capplets/mouse/gnome-settings-mouse.desktop.in.in
--- control-center-2.8.0.orig/capplets/mouse/gnome-settings-mouse.desktop.in.in	2003-12-16 22:38:15.000000000 +0100
+++ control-center-2.8.0/capplets/mouse/gnome-settings-mouse.desktop.in.in	2004-09-16 22:49:04.546047552 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/network/gnome-network-preferences.desktop.in.in control-center-2.8.0/capplets/network/gnome-network-preferences.desktop.in.in
--- control-center-2.8.0.orig/capplets/network/gnome-network-preferences.desktop.in.in	2004-04-15 20:32:22.000000000 +0200
+++ control-center-2.8.0/capplets/network/gnome-network-preferences.desktop.in.in	2004-09-16 22:49:19.076838536 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/sound/gnome-settings-sound.desktop.in.in control-center-2.8.0/capplets/sound/gnome-settings-sound.desktop.in.in
--- control-center-2.8.0.orig/capplets/sound/gnome-settings-sound.desktop.in.in	2003-12-16 22:38:50.000000000 +0100
+++ control-center-2.8.0/capplets/sound/gnome-settings-sound.desktop.in.in	2004-09-16 22:50:00.206585864 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/theme-switcher/gtk-theme-selector.desktop.in.in control-center-2.8.0/capplets/theme-switcher/gtk-theme-selector.desktop.in.in
--- control-center-2.8.0.orig/capplets/theme-switcher/gtk-theme-selector.desktop.in.in	2003-12-16 22:38:50.000000000 +0100
+++ control-center-2.8.0/capplets/theme-switcher/gtk-theme-selector.desktop.in.in	2004-09-16 22:50:14.395428832 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/ui-properties/gnome-ui-properties.desktop.in.in control-center-2.8.0/capplets/ui-properties/gnome-ui-properties.desktop.in.in
--- control-center-2.8.0.orig/capplets/ui-properties/gnome-ui-properties.desktop.in.in	2003-12-16 22:38:51.000000000 +0100
+++ control-center-2.8.0/capplets/ui-properties/gnome-ui-properties.desktop.in.in	2004-09-16 22:50:31.661803944 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
diff -ur control-center-2.8.0.orig/capplets/windows/window-properties.desktop.in.in control-center-2.8.0/capplets/windows/window-properties.desktop.in.in
--- control-center-2.8.0.orig/capplets/windows/window-properties.desktop.in.in	2003-12-16 22:38:52.000000000 +0100
+++ control-center-2.8.0/capplets/windows/window-properties.desktop.in.in	2004-09-16 22:50:45.692670928 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=control-center
