--- control-center-2.15.91/capplets/keybindings/gnome-keybinding-properties.c.orig	2006-08-15 18:56:34.000000000 -0400
+++ control-center-2.15.91/capplets/keybindings/gnome-keybinding-properties.c	2006-08-15 18:56:45.000000000 -0400
@@ -64,6 +64,28 @@
   { NULL }
 };
 
+#define COMPIZ_KEY(k) { "/apps/compiz/general/allscreens/options/" #k, ALWAYS_VISIBLE, 0 }
+
+static const KeyListEntry compiz_key_list[] =
+{
+  COMPIZ_KEY (close_window),
+  COMPIZ_KEY (lower_window),
+  COMPIZ_KEY (main_menu),
+  COMPIZ_KEY (maximize_window),
+  COMPIZ_KEY (maximize_window_horizontally),
+  COMPIZ_KEY (maximize_window_vertically),
+  COMPIZ_KEY (minimize_window),
+  COMPIZ_KEY (opacity_decrease),
+  COMPIZ_KEY (opacity_increase),
+  COMPIZ_KEY (show_desktop),
+  COMPIZ_KEY (toggle_window_maximized),
+  COMPIZ_KEY (toggle_window_maximized_horizontally),
+  COMPIZ_KEY (toggle_window_maximized_vertically),
+  COMPIZ_KEY (toggle_window_shaded),
+  COMPIZ_KEY (unmaximize_window),
+  COMPIZ_KEY (window_menu),
+};
+
 static const KeyListEntry metacity_key_list[] =
 {
   { "/apps/metacity/window_keybindings/activate_window_menu",      ALWAYS_VISIBLE,  0 },
@@ -559,6 +581,10 @@
     {
       append_keys_to_tree (dialog, _("Window Management"), metacity_key_list);
     }
+  else if (strcmp((char *) wm_name, WM_COMMON_COMPIZ) == 0)
+    {
+      append_keys_to_tree (dialog, _("Window Management"), compiz_key_list);
+    }
 }
 
 static void
--- control-center-2.15.91/capplets/common/wm-common.h.orig	2006-08-15 18:56:14.000000000 -0400
+++ control-center-2.15.91/capplets/common/wm-common.h	2006-08-15 18:56:45.000000000 -0400
@@ -2,6 +2,7 @@
 #define WM_COMMON_H
 
 #define WM_COMMON_METACITY "Metacity"
+#define WM_COMMON_COMPIZ   "compiz"
 #define WM_COMMON_SAWFISH  "Sawfish"
 #define WM_COMMON_UNKNOWN  "Unknown"
 
