--- control-center-1.4.0.1/capplets/background-properties/property-background.c.setroothint	Thu Aug 23 18:03:27 2001
+++ control-center-1.4.0.1/capplets/background-properties/property-background.c	Thu Aug 23 18:08:02 2001
@@ -164,6 +164,12 @@
 
 			tmp.pixel = xpixel_from_color (&state->bgColor1);
 			
+			XChangeProperty (GDK_DISPLAY(), GDK_ROOT_WINDOW(),
+					 gdk_atom_intern("_XROOTCOLOR_PIXEL", FALSE), 
+					 gdk_atom_intern("PIXEL", FALSE),
+					 32, PropModeReplace,
+					 (guchar *) &tmp.pixel, 1);
+
 			gdk_window_set_background (GDK_ROOT_PARENT(), &tmp);
 			gdk_window_clear (GDK_ROOT_PARENT());
 
