diff -Nuard control-center-1.4.0.5.orig/doc/C/control-center-C.omf control-center-1.4.0.5/doc/C/control-center-C.omf
--- control-center-1.4.0.5.orig/doc/C/control-center-C.omf	Fri Oct 26 11:00:10 2001
+++ control-center-1.4.0.5/doc/C/control-center-C.omf	Wed Oct  2 14:12:33 2002
@@ -1,15 +1,23 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
-      GNOME Control Center Manual
+	GNOME Control Center Manual
     </title>
-    <subject>
-      <category>System|Configuration|Configuration Tools</category>
-      <category>GNOME|Settings</category>
-    </subject>
+    <date>
+	2002-10-02
+    </date>
+    <version identifier="1.4.0.5" date="2002-10-02"/>
+    <subject category="System|Configuration|Configuration Tools"/>
+    <subject category="GNOME|Settings"/>
+    <description/>
+    <type/>
     <format mime="text/sgml"/>
     <identifier url="control-center.sgml"/>
     <language code="C"/>
+    <relation seriesid="0f71f7fe-d5fd-11d6-86b7-d475b9c461fe"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
diff -Nuard control-center-1.4.0.5.orig/doc/eu/control-center-eu.omf control-center-1.4.0.5/doc/eu/control-center-eu.omf
--- control-center-1.4.0.5.orig/doc/eu/control-center-eu.omf	Mon Feb 25 20:12:17 2002
+++ control-center-1.4.0.5/doc/eu/control-center-eu.omf	Wed Oct  2 14:14:02 2002
@@ -1,15 +1,23 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
-      GNOMEren Kontrol Zentroaren eskuliburua
+	GNOMEren Kontrol Zentroaren eskuliburua
     </title>
-    <subject>
-      <category>Sistema|Konfigurazioa|Konfigurazio-tresnak</category>
-      <category>GNOME|ezarpenak</category>
-    </subject>
+    <date>
+	2002-10-02
+    </date>
+    <version identifier="1.4.0.5" date="2002-10-02"/>
+    <subject category="Sistema|Konfigurazioa|Konfigurazio-tresnak"/>
+    <subject category="GNOME|ezarpenak"/>
+    <description/>
+    <type/>
     <format mime="text/sgml"/>
     <identifier url="control-center.sgml"/>
     <language code="eu"/>
+    <relation seriesid="0f71f7fe-d5fd-11d6-86b7-d475b9c461fe"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
