diff -urN aaa/control-center-2.3.4/gnome-settings-daemon/Makefile.am control-center-2.3.4/gnome-settings-daemon/Makefile.am
--- aaa/control-center-2.3.4/gnome-settings-daemon/Makefile.am	2003-06-29 21:39:32.000000000 +0200
+++ control-center-2.3.4/gnome-settings-daemon/Makefile.am	2003-08-01 01:32:46.000000000 +0200
@@ -52,7 +52,7 @@
 	$(GNOME_SETTINGS_DAEMON_LIBS)					\
 	$(top_builddir)/libbackground/libbackground.la			\
 	$(top_builddir)/libsounds/libsounds.a				\
-	-L$(top_builddir)/libwindow-settings/ -lgnome-window-settings 	\
+	$(top_builddir)/libwindow-settings/libgnome-window-settings.la 	\
 	$(XF86MISC_LIBS)
 
 #AccessX_files = \
