diff -urN control-center-2.11.92.orig/capplets/keyboard/Makefile.am control-center-2.11.92/capplets/keyboard/Makefile.am
--- control-center-2.11.92.orig/capplets/keyboard/Makefile.am	2005-08-22 22:44:26.558451264 +0200
+++ control-center-2.11.92/capplets/keyboard/Makefile.am	2005-08-22 22:46:46.310205768 +0200
@@ -29,6 +29,6 @@
 Desktop_in_files = keyboard.desktop.in
 desktop_DATA = $(Desktop_in_files:.desktop.in=.desktop)
 
-INCLUDES   = $(GNOMECC_CAPPLETS_CFLAGS) $(LIBXKLAVIER_CFLAGS)
+INCLUDES   = $(GNOMECC_CAPPLETS_CFLAGS) $(LIBXKLAVIER_CFLAGS) $(PANGOXFT_CFLAGS)
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES) $(Desktop_in_files) $(desktop_DATA)
 EXTRA_DIST = $(Glade_DATA) $(pixmaps_DATA)
diff -urN control-center-2.11.92.orig/configure.in control-center-2.11.92/configure.in
--- control-center-2.11.92.orig/configure.in	2005-08-22 22:44:27.329334072 +0200
+++ control-center-2.11.92/configure.in	2005-08-22 22:47:35.247766128 +0200
@@ -126,7 +126,9 @@
 
 PKG_CHECK_MODULES(AT_CAPPLET, $COMMON_MODULES)
 
-PKG_CHECK_MODULES(PANGOXFT, pangoxft >= 1.8)
+PKG_CHECK_MODULES(PANGOXFT, pangoxft >= 1.9.1)
+AC_SUBST(PANGOXFT_CFLAGS)
+AC_SUBST(PANGOXFT_LIBS)
 				
 dnl
 dnl Check for XRandR, needed for display capplet
