--- control-center-2.11.6/configure.in.wiget	2005-07-14 07:31:49.000000000 +0000
+++ control-center-2.11.6/configure.in	2005-07-14 07:33:52.000000000 +0000
@@ -19,7 +19,7 @@
 
 m4_define(gnomevfs_minver, 2.0.0)
 m4_define(fontconfig_minver, 1.0.0)
-m4_define(xft_minver, 2.1.7)
+m4_define(xft_minver, 2.1.2.2)
 m4_define(gtk_minver, 2.6.0)
 m4_define(libbonobo_minver, 2.0.0)
 m4_define(libgnomeui_minver, 2.2.0)
