diff -Nur control-center.orig/splash.c control-center/splash.c
--- control-center.orig/splash.c	Thu Feb  8 20:07:54 2001
+++ control-center/splash.c	Mon Aug 20 23:24:41 2001
@@ -171,6 +171,16 @@
 	  info[si_distribution] = g_strdup(buf);
 	  fclose(f);
 	}
+  } else if (g_file_exists("/etc/pld-release")) {
+    FILE *f;
+	gchar buf[80];
+
+	f = fopen("/etc/pld-release", "r");
+	if (f) { 
+	  fgets(buf, 79, f);
+	  info[si_distribution] = g_strdup(buf);
+	  fclose(f);
+	}
   }		
   else {
     /* This is about the only case where we'll put Unknown instead
