diff -urN control-center-2.1.0.1.orig/capplets/font/main.c control-center-2.1.0.1/capplets/font/main.c
--- control-center-2.1.0.1.orig/capplets/font/main.c	Mon Oct  7 00:57:43 2002
+++ control-center-2.1.0.1/capplets/font/main.c	Mon Oct  7 00:58:35 2002
@@ -9,6 +9,7 @@
 #include <gconf/gconf-client.h>
 #include <glade/glade.h>
 #include <stdarg.h>
+#include <fontconfig/fontconfig.h>
 
 #ifdef HAVE_XFT2
 #include <gdk/gdkx.h>
diff -urN control-center-2.1.0.1.orig/control-center/main.c control-center-2.1.0.1/control-center/main.c
--- control-center-2.1.0.1.orig/control-center/main.c	Mon Oct  7 00:57:43 2002
+++ control-center-2.1.0.1/control-center/main.c	Mon Oct  7 00:58:10 2002
@@ -32,6 +32,7 @@
 #include <X11/Xlib.h>
 #include <X11/Xatom.h>
 #include <gdk/gdkx.h>
+#include <fontconfig/fontconfig.h>
 
 #include "capplet-dir.h"
 #include "capplet-dir-view.h"
