diff -Nur control-center-1.99.8/Makefile.am control-center-1.99.8.new/Makefile.am
--- control-center-1.99.8/Makefile.am	Thu Apr 11 04:06:40 2002
+++ control-center-1.99.8.new/Makefile.am	Tue Apr 30 03:23:48 2002
@@ -1,4 +1,4 @@
-SUBDIRS = help po libsounds libbackground gnome-settings-daemon capplets control-center idl schemas
+SUBDIRS = po libsounds libbackground gnome-settings-daemon capplets control-center idl schemas
 
 #I removed the root-manager for now
 #
diff -Nur control-center-1.99.8/configure.in control-center-1.99.8.new/configure.in
--- control-center-1.99.8/configure.in	Mon Apr 29 21:45:06 2002
+++ control-center-1.99.8.new/configure.in	Tue Apr 30 03:23:38 2002
@@ -210,6 +210,4 @@
 capplets/accessibility/keyboard/Makefile
 capplets/network/Makefile
 schemas/Makefile
-help/Makefile
-help/C/Makefile
 libsounds/Makefile])
diff -Nur control-center-1.99.8/schemas/Makefile.am control-center-1.99.8.new/schemas/Makefile.am
--- control-center-1.99.8/schemas/Makefile.am	Tue Mar 26 18:22:22 2002
+++ control-center-1.99.8.new/schemas/Makefile.am	Tue Apr 30 03:15:49 2002
@@ -5,7 +5,4 @@
 
 schema_source_data = $(foreach file, $(schema_DATA), $(srcdir)/$(file))
 
-install-data-local: $(schema_installed_data)
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(schema_source_data)
-
 EXTRA_DIST=$(schema_DATA)
