diff -Nuard control-center-2.4.0.orig/capplets/accessibility/at-properties/Makefile.am control-center-2.4.0/capplets/accessibility/at-properties/Makefile.am
--- control-center-2.4.0.orig/capplets/accessibility/at-properties/Makefile.am	2003-06-25 19:37:32.000000000 +0200
+++ control-center-2.4.0/capplets/accessibility/at-properties/Makefile.am	2003-11-09 01:23:35.000000000 +0100
@@ -9,16 +9,12 @@
 
 @INTLTOOL_DESKTOP_RULE@
 
-iconsdir   = $(GNOMECC_ICONS_DIR)
-icons_DATA = at-enable-capplet.png
-
 desktopdir = $(GNOMECC_DESKTOP_DIR)
 Desktop_in_files = at-properties.desktop.in
 desktop_DATA = $(Desktop_in_files:.desktop.in=.desktop)
 
 pixmapdir  = $(GNOMECC_PIXMAPS_DIR)
 pixmap_DATA =					\
-	at-enable-capplet.png			\
 	at-startup.png				\
 	at-support.png
 
@@ -33,4 +29,4 @@
 	     -DGLADEDIR=\""$(gladedir)"\" \
 	     -DPIXMAPDIR=\""$(pixmapdir)"\"
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(icons_DATA) $(glade_DATA) $(pixmap_DATA) $(Desktop_in_files)
+EXTRA_DIST = $(glade_DATA) $(pixmap_DATA) $(Desktop_in_files)
diff -Nuard control-center-2.4.0.orig/capplets/accessibility/keyboard/Makefile.am control-center-2.4.0/capplets/accessibility/keyboard/Makefile.am
--- control-center-2.4.0.orig/capplets/accessibility/keyboard/Makefile.am	2002-06-18 20:55:04.000000000 +0200
+++ control-center-2.4.0/capplets/accessibility/keyboard/Makefile.am	2003-11-09 00:42:04.000000000 +0100
@@ -26,8 +26,8 @@
 Gladedir   = $(GNOMECC_GLADE_DIR)
 Glade_DATA = gnome-accessibility-keyboard-properties.glade
 
-iconsdir   = $(GNOMECC_ICONS_DIR)
-icons_DATA = accessibility-keyboard-capplet.png
+#iconsdir   = $(GNOMECC_ICONS_DIR)
+#icons_DATA = accessibility-keyboard-capplet.png
 
 desktop_iconsdir = $(datadir)/pixmaps
 desktop_icons_DATA = accessibility-keyboard-capplet.png
@@ -38,5 +38,5 @@
 
 INCLUDES   = $(GNOMECC_CAPPLETS_CFLAGS)
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(Glade_DATA) $(icons_DATA) $(Desktop_in_files) $(pixmap_DATA)
+EXTRA_DIST = $(Glade_DATA) $(Desktop_in_files) $(pixmap_DATA)
 
diff -Nuard control-center-2.4.0.orig/capplets/background/Makefile.am control-center-2.4.0/capplets/background/Makefile.am
--- control-center-2.4.0.orig/capplets/background/Makefile.am	2002-04-02 02:27:40.000000000 +0200
+++ control-center-2.4.0/capplets/background/Makefile.am	2003-11-09 00:41:28.000000000 +0100
@@ -8,8 +8,8 @@
 Gladedir   = $(GNOMECC_GLADE_DIR)
 Glade_DATA = background-properties.glade
 
-iconsdir   = $(GNOMECC_ICONS_DIR)
-icons_DATA = background-capplet.png
+#iconsdir   = $(GNOMECC_ICONS_DIR)
+#icons_DATA = background-capplet.png
 
 desktop_iconsdir = $(datadir)/pixmaps
 desktop_icons_DATA = background-capplet.png
@@ -20,5 +20,5 @@
 
 INCLUDES   = $(GNOMECC_CAPPLETS_CFLAGS) -I$(top_srcdir)/libbackground
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(Glade_DATA) $(icons_DATA) $(Desktop_in_files)
+EXTRA_DIST = $(Glade_DATA) $(Desktop_in_files)
 
diff -Nuard control-center-2.4.0.orig/capplets/default-applications/Makefile.am control-center-2.4.0/capplets/default-applications/Makefile.am
--- control-center-2.4.0.orig/capplets/default-applications/Makefile.am	2002-04-10 08:07:58.000000000 +0200
+++ control-center-2.4.0/capplets/default-applications/Makefile.am	2003-11-09 00:24:30.000000000 +0100
@@ -6,8 +6,8 @@
 
 @INTLTOOL_DESKTOP_RULE@
 
-iconsdir   = $(GNOMECC_ICONS_DIR)
-icons_DATA = default-applications-capplet.png
+#iconsdir   = $(GNOMECC_ICONS_DIR)
+#icons_DATA = default-applications-capplet.png
 
 desktop_iconsdir = $(datadir)/pixmaps
 desktop_icons_DATA = default-applications-capplet.png
@@ -21,4 +21,4 @@
 
 INCLUDES   = $(GNOMECC_CAPPLETS_CFLAGS)
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(Glade_DATA) $(icons_DATA) $(Desktop_in_files) gnome-default-applications-properties-structs.c
+EXTRA_DIST = $(Glade_DATA) $(Desktop_in_files) gnome-default-applications-properties-structs.c
diff -Nuard control-center-2.4.0.orig/capplets/desktop-links/Makefile.am control-center-2.4.0/capplets/desktop-links/Makefile.am
--- control-center-2.4.0.orig/capplets/desktop-links/Makefile.am	2002-12-06 11:53:21.000000000 +0100
+++ control-center-2.4.0/capplets/desktop-links/Makefile.am	2003-11-09 01:29:07.000000000 +0100
@@ -3,14 +3,13 @@
 
 ## Pixmaps
 
-pixmaps_pre = $(capplets:=-capplet.png)
-pixmaps_DATA = $(pixmaps_pre) accessibility-directory.png
-pixmapsdir = $(GNOMECC_ICONS_DIR)
-
 ##
 ##  Desktop files
 ##
 
+desktop_iconsdir = $(datadir)/pixmaps
+desktop_icons_DATA = accessibility-directory.png
+
 desktop_files = $(capplets:=.desktop)
 desktop_in_files = $(desktop_files:.desktop=.desktop.in)
 desktop_in_in_files = $(desktop_files:.desktop=.desktop.in.in)
@@ -19,10 +18,10 @@
 desktopfiles_DATA = $(desktop_files)
 
 CLEANFILES = $(desktop_files) $(desktop_in_files)
-EXTRA_DIST = $(desktop_in_in_files) $(groups_in_in_files) $(pixmaps_DATA)
+EXTRA_DIST = $(desktop_in_in_files) $(groups_in_in_files)
 
 %.desktop.in: $(srcdir)/%.desktop.in.in
-	    sed s#Icon=#Icon=$(GNOMECC_ICONS_DIR)/# < $< > $@
+	    sed s#Icon=#Icon=# < $< > $@
 
 @INTLTOOL_DESKTOP_RULE@
 
@@ -46,6 +45,6 @@
 Gnomecc_DATA = $(groups_files)
 
 %.directory.in: $(srcdir)/%.directory.in.in
-	    sed s#Icon=#Icon=$(pixmapsdir)/# < $< > $@
+	    sed s#Icon=#Icon=# < $< > $@
 
 @INTLTOOL_DIRECTORY_RULE@
diff -Nuard control-center-2.4.0.orig/capplets/display/Makefile.am control-center-2.4.0/capplets/display/Makefile.am
--- control-center-2.4.0.orig/capplets/display/Makefile.am	2003-04-23 17:01:16.000000000 +0200
+++ control-center-2.4.0/capplets/display/Makefile.am	2003-11-09 00:28:11.000000000 +0100
@@ -7,8 +7,8 @@
 
 @INTLTOOL_DESKTOP_RULE@
 
-iconsdir   = $(GNOMECC_ICONS_DIR)
-icons_DATA = display-capplet.png
+#iconsdir   = $(GNOMECC_ICONS_DIR)
+#icons_DATA = display-capplet.png
 
 desktop_iconsdir = $(datadir)/pixmaps
 desktop_icons_DATA = display-capplet.png
@@ -23,4 +23,4 @@
 	     -DPIXMAPDIR=\""$(pixmapdir)"\"
 
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(Desktop_in_files) $(icons_DATA)
+EXTRA_DIST = $(Desktop_in_files)
diff -Nuard control-center-2.4.0.orig/capplets/font/Makefile.am control-center-2.4.0/capplets/font/Makefile.am
--- control-center-2.4.0.orig/capplets/font/Makefile.am	2002-07-24 19:36:30.000000000 +0200
+++ control-center-2.4.0/capplets/font/Makefile.am	2003-11-09 00:29:04.000000000 +0100
@@ -7,8 +7,8 @@
 
 @INTLTOOL_DESKTOP_RULE@
 
-iconsdir   = $(GNOMECC_ICONS_DIR)
-icons_DATA = font-capplet.png
+#iconsdir   = $(GNOMECC_ICONS_DIR)
+#icons_DATA = font-capplet.png
 
 pixmapdir  = $(GNOMECC_PIXMAPS_DIR)
 pixmap_DATA =					\
@@ -32,4 +32,4 @@
 	     -DGLADEDIR=\""$(gladedir)"\" \
 	     -DPIXMAPDIR=\""$(pixmapdir)"\"
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(Desktop_in_files) $(icons_DATA) $(glade_DATA) $(pixmap_DATA)
+EXTRA_DIST = $(Desktop_in_files) $(glade_DATA) $(pixmap_DATA)
diff -Nuard control-center-2.4.0.orig/capplets/keybindings/Makefile.am control-center-2.4.0/capplets/keybindings/Makefile.am
--- control-center-2.4.0.orig/capplets/keybindings/Makefile.am	2002-08-13 20:14:39.000000000 +0200
+++ control-center-2.4.0/capplets/keybindings/Makefile.am	2003-11-09 00:43:28.000000000 +0100
@@ -14,8 +14,8 @@
 Gladedir   = $(GNOMECC_GLADE_DIR)
 Glade_DATA = gnome-keybinding-properties.glade
 
-pixmapsdir = $(datadir)/pixmaps
-pixmaps_DATA = keyboard-shortcut.png
+desktop_iconsdir = $(datadir)/pixmaps
+desktop_icons_DATA = keyboard-shortcut.png
 
 desktopdir = $(GNOMECC_DESKTOP_DIR)
 Desktop_in_files = keybinding.desktop.in
@@ -23,5 +23,5 @@
 
 INCLUDES   = $(GNOMECC_CAPPLETS_CFLAGS)
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(Glade_DATA) $(Desktop_in_files) $(pixmaps_DATA)
+EXTRA_DIST = $(Glade_DATA) $(Desktop_in_files)
 
diff -Nuard control-center-2.4.0.orig/capplets/keyboard/Makefile.am control-center-2.4.0/capplets/keyboard/Makefile.am
--- control-center-2.4.0.orig/capplets/keyboard/Makefile.am	2002-04-10 08:08:02.000000000 +0200
+++ control-center-2.4.0/capplets/keyboard/Makefile.am	2003-11-09 00:45:04.000000000 +0100
@@ -11,14 +11,13 @@
 Glade_DATA = gnome-keyboard-properties.glade
 
 iconsdir   = $(GNOMECC_PIXMAPS_DIR)
-icons_DATA = keyboard-capplet.png	\
-	keyboard-repeat.png		\
+icons_DATA = keyboard-repeat.png		\
 	keyboard-cursor.png		\
 	keyboard-volume.png		\
 	keyboard-bell.png
 
-pixmapsdir = $(datadir)/pixmaps
-pixmaps_DATA = keyboard-capplet.png
+desktop_iconsdir = $(datadir)/pixmaps
+desktop_icons_DATA = keyboard-capplet.png
 
 desktopdir = $(GNOMECC_DESKTOP_DIR)
 Desktop_in_files = keyboard.desktop.in
diff -Nuard control-center-2.4.0.orig/capplets/mouse/Makefile.am control-center-2.4.0/capplets/mouse/Makefile.am
--- control-center-2.4.0.orig/capplets/mouse/Makefile.am	2003-01-08 08:04:42.000000000 +0100
+++ control-center-2.4.0/capplets/mouse/Makefile.am	2003-11-09 00:45:23.000000000 +0100
@@ -27,9 +27,6 @@
 Gladedir   = $(GNOMECC_GLADE_DIR)
 Glade_DATA = gnome-mouse-properties.glade
 
-iconsdir   = $(GNOMECC_ICONS_DIR)
-icons_DATA = mouse-capplet.png
-
 desktop_iconsdir = $(datadir)/pixmaps
 desktop_icons_DATA = mouse-capplet.png
 
@@ -39,4 +36,4 @@
 
 INCLUDES   = $(GNOMECC_CAPPLETS_CFLAGS)
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(Glade_DATA) $(icons_DATA) $(Desktop_in_files) $(pixmap_DATA) $(cursorfont_DATA)
+EXTRA_DIST = $(Glade_DATA) $(Desktop_in_files) $(pixmap_DATA) $(cursorfont_DATA)
diff -Nuard control-center-2.4.0.orig/capplets/ui-properties/Makefile.am control-center-2.4.0/capplets/ui-properties/Makefile.am
--- control-center-2.4.0.orig/capplets/ui-properties/Makefile.am	2002-04-10 08:08:05.000000000 +0200
+++ control-center-2.4.0/capplets/ui-properties/Makefile.am	2003-11-09 00:46:57.000000000 +0100
@@ -6,9 +6,6 @@
 
 @INTLTOOL_DESKTOP_RULE@
 
-iconsdir   = $(GNOMECC_ICONS_DIR)
-icons_DATA = behavior-capplet.png
-
 desktop_iconsdir = $(datadir)/pixmaps
 desktop_icons_DATA = behavior-capplet.png
 
@@ -19,4 +16,4 @@
 Glade_DATA = gnome-ui-properties.glade
 INCLUDES   = $(GNOMECC_CAPPLETS_CFLAGS)
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(Glade_DATA) $(icons_DATA) $(Desktop_in_files)
+EXTRA_DIST = $(Glade_DATA) $(Desktop_in_files)
diff -Nuard control-center-2.4.0.orig/capplets/windows/Makefile.am control-center-2.4.0/capplets/windows/Makefile.am
--- control-center-2.4.0.orig/capplets/windows/Makefile.am	2002-10-29 07:19:51.000000000 +0100
+++ control-center-2.4.0/capplets/windows/Makefile.am	2003-11-09 00:38:50.000000000 +0100
@@ -14,9 +14,6 @@
 gladedir   = $(GNOMECC_GLADE_DIR)
 glade_DATA = gnome-window-properties.glade
 
-iconsdir   = $(GNOMECC_ICONS_DIR)
-icons_DATA = window-capplet.png
-
 desktop_iconsdir = $(datadir)/pixmaps
 desktop_icons_DATA = window-capplet.png
 
@@ -31,5 +28,5 @@
 	     -DPIXMAPDIR=\""$(pixmapdir)"\"
 
 CLEANFILES = $(GNOMECC_CAPPLETS_CLEANFILES)
-EXTRA_DIST = $(glade_DATA) $(icons_DATA) $(Desktop_in_files) $(pixmap_DATA)
+EXTRA_DIST = $(glade_DATA) $(Desktop_in_files) $(pixmap_DATA)
 
diff -Nuard control-center-2.4.0.orig/control-center/Makefile.am control-center-2.4.0/control-center/Makefile.am
--- control-center-2.4.0.orig/control-center/Makefile.am	2003-05-07 20:55:08.000000000 +0200
+++ control-center-2.4.0/control-center/Makefile.am	2003-11-08 23:49:14.000000000 +0100
@@ -15,7 +15,6 @@
 
 INCLUDES = 							\
 	-DGNOMELOCALEDIR=\""$(prefix)/$(DATADIRNAME)/locale"\" 	\
-	-DSETTINGS_DIR=\""$(GNOMECC_DESKTOP_DIR)"\"		\
 	-DGNOMECC_ICONS_DIR=\""$(GNOMECC_ICONS_DIR)"\"		\
 	-DPIXMAP_DIR=\""$(datadir)/pixmaps"\"			\
 	-DART_DIR=\""$(splashdir)"\"				\
