diff -urN aa/control-center-2.8.0/capplets/default-applications/gnome-default-applications-properties-structs.c control-center-2.8.0/capplets/default-applications/gnome-default-applications-properties-structs.c
--- aa/control-center-2.8.0/capplets/default-applications/gnome-default-applications-properties-structs.c	2004-06-25 22:52:55.000000000 +0200
+++ control-center-2.8.0/capplets/default-applications/gnome-default-applications-properties-structs.c	2004-09-15 12:38:02.736663944 +0200
@@ -31,7 +31,7 @@
 	 * there is only one (the first entry in this list) Evolution entry 
 	 * in the list shown to the user
 	 */
-	{ N_("Evolution Mail Reader"),		"evolution-1.6",	"evolution-1.6 %s",	FALSE, FALSE, },
+	{ N_("Evolution Mail Reader"),		"evolution-2.0",	"evolution-2.0 %s",	FALSE, FALSE, },
 	{ N_("Evolution Mail Reader"),		"evolution-1.5",	"evolution-1.5 %s",	FALSE, FALSE, },
         { N_("Evolution Mail Reader"),		"evolution-1.4",	"evolution-1.4 %s",	FALSE, FALSE, },
         { N_("Evolution Mail Reader"),		"evolution",		"evolution %s",		FALSE, FALSE, },
