diff -Nru control-center-1.2.1/capplets/background-properties/property-background.c control-center-1.2.1.new/capplets/background-properties/property-background.c
--- control-center-1.2.1/capplets/background-properties/property-background.c	Mon Aug  7 20:02:14 2000
+++ control-center-1.2.1.new/capplets/background-properties/property-background.c	Mon Aug  7 20:10:23 2000
@@ -590,7 +590,10 @@
 		else if (origState.wpFileName)
 			gtk_file_selection_set_filename (GTK_FILE_SELECTION (fileSel),
 							 origState.wpFileName);
-	
+		else if (g_file_exists(GNOME_ICONDIR"/backgrounds"))
+			gtk_file_selection_set_filename (GTK_FILE_SELECTION (fileSel),
+							GNOME_ICONDIR"/backgrounds");
+
 		gtk_signal_connect (GTK_OBJECT (fileSel), "delete_event",
 				    (GtkSignalFunc) delete_browse,
 				    &fileSel);
